src/LoRa.h

```diff
@@ -66,11 +66,16 @@ class LoRaClass : public Stream {
   void setSyncWord(int sw);
   void enableCrc();
   void disableCrc();
+  void enableInvertIQ();
+  void disableInvertIQ();
+  
   void setOCP(uint8_t mA); // Over Current Protection control
 
   // deprecated
   void crc() { enableCrc(); }
   void noCrc() { disableCrc(); }
+  void invertIQ() { enableInvertIQ(); }
+  void noInvertIQ() { disableInvertIQ(); }
 
```

Comment:
I'd say we remove `invertIQ()` and `noInvertIQ()` instead of adding deprecated methods.

Reply:
sir is it possible to communicate  RN2483 with sx1276 we tried but data not receiving in rn2483 but sx1276 receives data from rn2483 is it because of serial and spi communication